code/game/objects/items/weapons/storage/firstaid.dm

```diff
@@ -9,8 +9,8 @@
  * First Aid Kits
  */
 /obj/item/storage/firstaid
-	name = "first-aid kit"
-	desc = "It's an emergency medical kit for those serious boo-boos."
+	name = "generic first-aid kit"
+	desc = "If you can see this, make a bug report on GitHub, something went wrong!"
 	icon_state = "firstaid"
```

Comment:
Hang on, will this still be printable from the autolathe and stuff? If it doesn't appear in-game at all, it might be worth changing its icon_state to something that will get a mapper's attention, or adding it to some of the CI checks we do to prevent it from getting mapped in.

Reply:
Took me 2 weeks but it's done now. A generic icon added.
![image](https://github.com/user-attachments/assets/90b01193-47ed-4e1a-aff0-7bd1b6833f20)
And 2 of them in sdmm
![image](https://github.com/user-attachments/assets/7971fd22-a1e8-43dd-a9b9-b5e283ea4b11)
`/obj/item/storage/firstaid` doesn't appear in autolathe or anywhere else. It's only created by mappers who place them on dmm files. No any other way to get them. So i hope this icon they will notice.